src/Evolution/Systems/ScalarWave/Tags.hpp

```diff
@@ -104,5 +105,33 @@ struct EvolvedFieldsFromCharacteristicFields : db::SimpleTag {
     return "EvolvedFieldsFromCharacteristicFields";
   }
 };
+
+/// The energy density of the scalar wave system
+template <size_t SpatialDim>
+struct EnergyDensity : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+
+/// @{
```

Comment:
(I think) you can use this to combine the documentation for the `EnergyDensity` and `EnergyDensityCompute` by moving it up to line 108 and then closing it after `EnergyDensityCompute`